react-ui/src/pages/contest/Contest.jsx

```diff
@@ -234,21 +71,15 @@ function Contest() {
   const { state = {} } = useLocation();
   const [isLoaded, setLoaded] = useState(false);
   const [selectedCategories, setSelectedCategories] = useState(state?.selectedCategories ?? []);
-  const [descriptionEntryId, setDescriptionEntryId] = useState(null);
-  const [isSettingsOpen, setSettingsOpen] = useState(false);
+  const [isInfoOpen, setInfoOpen] = useState(false);
```

Comment:
I believe at the time I thought settings was vague because there are profile settings and then display settings. But it looks like I did not fully commit either way. Lmk if you have a suggest to separate or if you prefer I revert the naming.

Reply:
How about `const [isDrawerOpen, setDrawerOpen]` so it no longer matters what we put in it?